scrapers/CopyToGallery.py

```diff
@@ -0,0 +1,151 @@
+import json
+import sys
+import os
+import requests
+
+import py_common.graphql as graphql
+import py_common.log as log
+
+SERVER_IP = "http://localhost:9999"
+# API key (Settings > Configuration > Authentication)
+STASH_API = ""
+SERVER_URL = SERVER_IP + "/graphql"
+
+find_gallery = False
+
+def call_graphql(query, variables=None):
+    headers = {
+        "Accept-Encoding": "gzip, deflate, br",
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+        "Connection": "keep-alive",
+        "DNT": "1",
+        "ApiKey": STASH_API
+    }
+    json = {'query': query}
+    if variables is not None:
+        json['variables'] = variables
+    try:
+        response = requests.post(SERVER_URL, json=json, headers=headers)
+        if response.status_code == 200:
+            result = response.json()
+            if result.get("error"):
+                for error in result["error"]["errors"]:
+                    raise Exception("GraphQL error: {}".format(error))
+            if result.get("data"):
+                return result.get("data")
+        elif response.status_code == 401:
+            log.debug("401 returned from callGraphQL")
+            # debug("[ERROR][GraphQL] HTTP Error 401, Unauthorised.")
+            return None
+        else:
+            raise ConnectionError("GraphQL query failed:{} - {}".format(response.status_code, response.content))
+    except Exception as err:
+        log.error(err)
+        # debug(err)
+        return None
+
+def get_gallery_id_by_path(gallery_path):
+    log.debug("get_gallery_by_path gallery_path " + str(gallery_path))
+    query = """
+            query FindGalleries($galleries_filter: GalleryFilterType) {
+              findGalleries(gallery_filter: $galleries_filter filter: {per_page: -1}) {
+                count
+                    galleries {
+                        id
+                    }
+              }
+            }
+            """
+    variables = {"galleries_filter": {"path": {'value': gallery_path, "modifier": "INCLUDES_ALL"}}}
+    result = call_graphql(query, variables)
+    log.debug("get_gallery_by_path callGraphQL result " + str(result))
+    return result['findGalleries']['galleries'][0]['id']
+
+def update_gallery(input):
+    log.debug("gallery input " + str(input))
+    query = """
+                mutation GalleryUpdate($input : GalleryUpdateInput!) {
+                  galleryUpdate(input: $input) {
+                    id
+                    title
+                  }
+                }
+                """
+    variables = {
+        "input": input
+    }
+    result = call_graphql(query, variables)
+    log.debug("updateGallery callGraphQL result " + str(result))

```

Comment:
As the scraper doesnt print anything by default i think we can switch this to `info` instead of debug so that the user is notified which galleries were updated.
Maybe make the print a little more user friendly also ? eg
```python
    if result:
        g_id = result['galleryUpdate'].get('id')
        g_title = result['galleryUpdate'].get('title')
        log.info(f"updated Gallery ({g_id}): {g_title}")
```

Reply:
No problem.